scripts/build

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+exec mkdocs build
```

Comment:
Why are the commands prefixed with `exec`? They should work fine without it.

Reply:
To ensure that their exit codes are passed back to the originator. I _think_ `bash` will always do that, however it's a little bit magical and I'm not sure whether it's consistent between shells.

It also has the benefit of avoiding launching a fresh process.